advocacy_docs/edb-postgres-ai/databases/database_operations/resource_monitoring/cpu_monitoring.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Basic CPU monitoring 
+navTitle: CPU monitoring
+description: Perform basic monitoring of your CPU metrics. 
+---
+
+CPU usage is essential for the processing of queries, connections, transactions and background processes. Therefore, it is important to ensure your CPU configuration can cover the requirements of your applications, and it has enough buffer to continue working through workload peeks.
+
+## Healthy CPU 
+
+As a general rule, a database server is considered to have a healthy CPU configuration when the average **CPU** usage is stable and does not surpass the 80% threshold of the allotted CPU resources over a longer period of time. This ensures the database server isn't overburdened.
```

Comment:
I'm not sure we should be making such a big deal of these 80% and 90% thresholds, they don't really have any significant other than just being round numbers.